int16 temperature_range_min
int16 temperature_range_max
---
bool success
